Formula/retdec.rb

```diff
@@ -0,0 +1,37 @@
+class Retdec < Formula
+  desc "Retargetable machine-code decompiler based on LLVM"
+  homepage "https://github.com/avast/retdec"
+  license all_of: ["MIT", "Zlib"]
+  head "https://github.com/avast/retdec.git", branch: "master"
+
+  stable do
+    url "https://github.com/avast/retdec.git",
+      branch:   "master",
+      revision: "53e55b4b26e9b843787f0e06d867441e32b1604e"
+    version "5.0"
```

Comment:
It will need a stable tag, not pointing to a branch

Reply:
Update with tag instead of branch